easyrsa3/easyrsa

```diff
@@ -760,7 +778,11 @@ $EASYRSA_EXTRA_EXTS"
 	# generate request
 	[ $EASYRSA_BATCH ] && opts="$opts -batch"
 	# shellcheck disable=2086,2148
-	easyrsa_openssl req -utf8 -new -newkey "$EASYRSA_ALGO":"$EASYRSA_ALGO_PARAMS" \
+	algo_opts=""
+	if [ "ed" != $EASYRSA_ALGO ];then
+		algo_opts=' -newkey $EASYRSA_ALGO:$EASYRSA_ALGO_PARAMS '
```

Comment:
Replace with double quotes (`""`). This is why the tests are failing (bash don't replace $EASYRSA_ALGO with single quotes (`''`))